cdap-docs/developers-manual/source/building-blocks/spark-programs.rst

```diff
@@ -4,17 +4,17 @@
 
 .. _spark:
 
-=============================================
-Spark Programs *(Beta, Standalone CDAP only)*
-=============================================
+==============
+Spark Programs
+==============
+
+.. rubric:: Beta, on Standalone and insecure Clusters only
```

Comment:
It sounds confusing to be "on Standalone and insecure Clusters only"
should we say supported on/runs on or is it too verbose ?